### DISCUSSION TITLE
How to use this for front end ?

### DISCUSSION THREAD

**Discussion by magic-thomas** [Power-Components/livewire-powergrid#265] [General]
I think it is a great package for backend side.

But I would like to use this package for client side. 

I wonder if it is secure or what to check  to make it secure. 

And I wonder how to use for categorized Posts Table. 

for example, What if I want to extract data from posts table  following with category_id which is given by url query string, not choosing by user. 

Does somebody have simple idea about this ?

**Comment by magic-thomas** [comment]
I found clue. 

Don't mind my question.  

Sorry for idiot question.

**Reply by JF-626164** [reply]
There's no such thing as an idiotic question my friend. Always ask, if you have a question about it others do too.
